docs/api-guide/throttling.md

```diff
@@ -92,6 +92,10 @@ If you need to use a cache other than `'default'`, you can do so by creating a c
 
 You'll need to remember to also set your custom throttle class in the `'DEFAULT_THROTTLE_CLASSES'` settings key, or using the `throttle_classes` view attribute.
 
+## A note on concurrency
+
+Since the default implementations of throttles are based on the Django cache system, race conditions in which concurrent requests may be able to bypass throttling are possible.  If your project requires more rock-solid throttles even in concurrent situations, you may need to implement such a throttle by hand. See [this issue][gh5181] for details.
```

Comment:
> race conditions in which concurrent requests may be able to bypass throttling are possible.

I might rewrite this as:

> concurrent requests may be able to bypass throttling due to race conditions.

Also, 

> you may need to implement such a throttle by hand.

Are there any 3rd party implementations we could point to?

Reply:
Rewritten, thank you.

As for those implementations... no idea, sorry :)